1D HDF5 Build Fails

Seen by @kale-j in https://github.com/ECP-WarpX/WarpX/issues/4915

AMReX with 1D Spacedim cannot be built with HDF5 enabled.
```
 [ 49%] Building CUDA object _deps/fetchedamrex-build/Src/CMakeFiles/amrex_1d.dir/Extern/HDF5/AMReX_PlotFileUtilHDF5.cpp.o
/global/homes/k/kjw/src/warpx/build_noqed/_deps/fetchedamrex-src/Src/Extern/HDF5/AMReX_ParticleHDF5.H(1203): error: no instance of constructor "amrex::Box::Box" matches the argument list
argument types are: ({...}, {...})
Box tmp ({boxes[i1]}, {boxes[i1*2]});
^

1 error detected in the compilation of "/global/homes/k/kjw/src/warpx/build_noqed/_deps/fetchedamrex-src/Src/Particle/AMReX_TracerParticles.cpp".
gmake[2]: *** [_deps/fetchedamrex-build/Src/CMakeFiles/amrex_1d.dir/build.make:1546: _deps/fetchedamrex-build/Src/CMakeFiles/amrex_1d.dir/Particle/AMReX_TracerParticles.cpp.o] Error 2
gmake[2]: *** Waiting for unfinished jobs....
gmake[1]: *** [CMakeFiles/Makefile2:1246: _deps/fetchedamrex-build/Src/CMakeFiles/amrex_1d.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
```

--- a/Src/Extern/HDF5/AMReX_ParticleHDF5.H
+++ b/Src/Extern/HDF5/AMReX_ParticleHDF5.H
@@ -1200,7 +1200,8 @@ ParticleContainer_impl<ParticleType, NArrayReal, NArrayInt, Allocator, CellAssig
         /* particle_box_arrays[lev].readFrom(phdr_file); */
         particle_box_arrays[lev] = ParticleBoxArray(lev);
         for (int i = 0; i < (int)ngrid; i++) {
-            Box tmp ({AMREX_D_DECL(boxes[i*AMREX_SPACEDIM], boxes[i*AMREX_SPACEDIM+1], boxes[i*AMREX_SPACEDIM+2])}, {AMREX_D_DECL(boxes[i*AMREX_SPACEDIM*2], boxes[i*AMREX_SPACEDIM*2+1], boxes[i*AMREX_SPACEDIM*2+2])});
+            Box tmp (IntVect{AMREX_D_DECL(boxes[i*AMREX_SPACEDIM], boxes[i*AMREX_SPACEDIM+1], boxes[i*AMREX_SPACEDIM+2])},
+                     IntVect{AMREX_D_DECL(boxes[i*AMREX_SPACEDIM*2], boxes[i*AMREX_SPACEDIM*2+1], boxes[i*AMREX_SPACEDIM*2+2])});
             particle_box_arrays[lev][i] = tmp;
         }
 

